src/custom/utils/analytics/index.ts

```diff
@@ -60,6 +60,11 @@ export function onChainIdChange(chainId: number | undefined) {
   ReactGA.set({ cd1: chainId ?? 0 })
 }
 
+export function onWalletChange(walletName: string | undefined) {
+  // cd2 - custom dimension 2 - walletName
+  ReactGA.set({ cd2: walletName })
```

Comment:
Is the name `cd2` defined by us in the GA console or is that pre-defined?

Reply:
I think its not a pre-defined name, its just how uniswap does it. I can't find why they want to do it like that.


To my knowledge, we can give it any name we want. For this reason, I would be happy to get read of cd1, cd2, .... and call them actually something meaningful 


<img width="1469" alt="image" src="https://user-images.githubusercontent.com/2352112/177188420-12ce5494-3409-4233-a017-46e5eac5a7de.png">


Add your custom dimensions here https://analytics.google.com/analytics/web/#/p316521862/customdefinitions/hub?params=_u..nav%3Dmaui%26_u..pageSize%3D25

More info https://support.google.com/analytics/answer/10075209?hl=en#zippy=%2Cin-this-article

Reply:
Updated to `walletName`